### ISSUE TITLE
Custom table for 

### ISSUE THREAD

**Issue by madanpraba** [soflyy/oxygen-bugs-and-features#2199]
**Describe the feature you'd like to see included in Oxygen.**
*A clear, detailed description of the feature you're requesting.*

**What are the use cases for this feature?**
*e.g. Accessibility, e-commerce store that sells bicycles, etc.*

**Examples of this feature or functionality.**
*If another product has done it well, provide examples here.*

**Comment by madanpraba** [comment]
Custom table for the post meta. For each post, Oxygen created meta value 'ct_', which created many rows. On a big site, this feature adds a lot of bloats. Kindly consider storing the value in a custom table similar to the Metabox plugin.

**Comment by madanpraba** [comment]
No, it is stored in wp_postmeta. Nearly 30 rows for each post. 



edit | 600 | 35 | ct_builder_shortcodes_revisions_dates | 
edit | 599 | 35 | ct_builder_shortcodes_revisions | 
edit | 598 | 35 | ct_builder_shortcodes_revisions_dates | 1622176042
edit | 597 | 35 | ct_builder_shortcodes_revisions | [
edit | 596 | 35 | ct_builder_shortcodes_revisions_dates | 1622175455
edit | 595 | 35 | ct_builder_shortcodes_revisions | [
edit | 594 | 35 | ct_builder_shortcodes_revisions_dates | 1622175446
edit | 593 | 35 | ct_builder_shortcodes_revisions |
edit | 592 | 36 | oxygen_lock_post_edit_mode | NULL
edit | 591 | 36 | ct_builder_shortcodes | &nbsp;
edit | 590 | 36 | ct_template_order | 0
edit | 589 | 36 | ct_template_apply_if_post_of_parents | &nbsp;
edit | 588 | 36 | ct_template_post_of_parents | a:0:{}
edit | 587 | 36 | ct_use_template_taxonomies | &nbsp;
edit | 586 | 36 | ct_template_taxonomies | a:2:{s:5:"names";a:0:{}s:6:"values";a:0:{}}
edit | 585 | 36 | ct_template_include_ids | &nbsp;
edit | 584 | 36 | ct_template_exclude_ids | &nbsp;
edit | 583 | 36 | ct_template_post_types | a:0:{}
edit | 582 | 36 | ct_template_single_all | &nbsp;
edit | 581 | 36 | ct_template_apply_if_archive_among_authors | &nbsp;
edit | 580 | 36 | ct_template_authors_archives | a:0:{}
edit | 579 | 36 | ct_template_apply_if_archive_among_cpt | &nbsp;
edit | 578 | 36 | ct_template_archive_post_types | a:0:{}
edit | 577 | 36 | ct_template_apply_if_archive_among_taxonomies | &nbsp;
edit | 576 | 36 | ct_template_archive_among_taxonomies | a:0:{}
edit | 575 | 36 | ct_template_all_archives | &nbsp;
edit | 574 | 36 | ct_template_404_page | &nbsp;
edit | 573 | 36 | ct_template_inner_content | &nbsp;
edit | 572 | 36 | ct_template_search_page | &nbsp;
edit | 571 | 36 | ct_template_date_archive | &nbsp;
edit | 570 | 36 | ct_template_blog_posts | &nbsp;
edit | 569 | 36 | ct_template_front_page | &nbsp;
edit | 568 | 36 | ct_template_index | &nbsp;
edit | 567 | 36 | ct_template_authors_archives_all | &nbsp;
edit | 566 | 36 | ct_template_custom_taxonomies_all | &nbsp;
edit | 565 | 36 | ct_template_custom_taxonomies | a:0:{}
edit | 564 | 36 | ct_template_tags_all | &nbsp;
edit | 563 | 36 | ct_template_tags | a:0:{}
edit | 562 | 36 | ct_template_categories_all | &nbsp;
edit | 561 | 36 | ct_template_categories | a:0:{}
edit | 560 | 36 | ct_template_archive_post_types_all | &nbsp;
edit | 559 | 36 | ct_other_template | 0
edit | 558 | 35 | ct_builder_shortcodes_revisions_dates | 1622175322
edit | 557 | 35 | ct_builder_shortcodes_revisions | [
edit | 556 | 35 | ct_builder_shortcodes_revisions_dates | 1622175308
edit | 555 | 35 | ct_builder_shortcodes_revisions

**Comment by wpsumo** [comment]
Sorry yes meant wp_postmeta. Ok yes I agree storing it in a custom table could also speed up and make it less complicated once Oxygen reads and write to the db while loading the builder UI.

**Comment by alexrollin** [comment]
+1
